src/ui/Loghandling/BinLogParser.h

```diff
@@ -76,6 +76,12 @@ class BinLogParser : public LogParserBase
     static const int s_FMTFormatSize = 16;       /// Size of the format field in FMT message
     static const int s_FMTLabelsSize = 64;       /// Size of the comma delimited names field in FMT message
 
+    static const quint32 s_FloatNaNDetector     = 0x7FC00000;   /// Value to mask and detect all float NaN from ardupilot
+    static const quint32 s_FloatSoftNaNDetector = 0x7FC04152;   /// Value to detect a quiet/soft float NaN from ardupilot
```

Comment:
These are, indeed, the only magic numbers that ArduPilot currently produces.  They are AP specific, kinda, in that one says "AP" and the other "ARDU" IIRC.

If another autopilot were to add their own soft nans then this check would need to be changed.

I like the specific check as it currently is, just mentioning this fact.

Reply:
I think we should change it as soon as there is another autopilot :wink:
Thanks a lot for that review!